src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImplImpl.java

```diff
@@ -0,0 +1,19 @@
+package com.dev.cinema.dao.impl;
```

Comment:
i think you can rename this class from `CinemaHallDaoImplImpl` to `CinemaHallDaoImpl`